.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
-target/*
\ No newline at end of file
+target/*
+/src/main/resources/
```

Comment:
do we need this folder here, why?

Reply:
No) In scope of these project we don't need, accidentally added it to `gitignore`.